src/Common/Auth/AuthUtils.php

```diff
@@ -1376,6 +1376,7 @@ public static function setUserSessionVariables($username, $hash, $userInfo, $aut
         $_SESSION['authUserID'] = $userInfo['id']; // user id
         $_SESSION['authProvider'] = $authGroup; // user group
         $_SESSION['userauthorized'] = $userInfo['authorized']; // user authorized setting
+        $_SESSION['seeAuth'] = $userInfo['see_auth']; // user see_auth setting (see auth group)
```

Comment:
Okay so now we have a session flag where is it implemented? If we need the flag it is retrieved when the user is fetched and used in place. I don't get why we need another var in session if not used unless needed for modules.

I thought see auth wasn't working and this fixed!! Or am I missing something?

Reply:
It is not implemented anywhere. That is why I added it. I used it to hide the provider list on the calendar so the provider only sees their own calendar and cannot view other providers calendars. That was the request.

Here is how I used it in the views/day/ajax_template.html

	 // EOS FF
	 // ==============================
	 echo "</div>";
		if ($_SESSION['see_auth'] != '2') {
	 echo "   <select multiple size='5' name='pc_username[]' id='pc_username' class='view2 form-control'>\n";
	 echo "    <option value='__PC_ALL__'>"  . xlt("All Users") . "</option>\n";
	 foreach ($provinfo as $doc) {
		$username = $doc['username'];
		echo "    <option value='" . attr($username) . "'";
		foreach ($providers as $provider)
			if ($provider['username'] == $username) echo " selected";
		echo ">" . text($doc['lname']) . ", " . text($doc['fname']) . "</option>\n";
	 }
	 echo "   </select>\n";
		}
	[-/php-]

Reply:
okay I guess I'm missing it because I can't find so I'm guessing you haven't implemented yet or my ide search engine is failing me?

Reply:
@sjpadgett, that code snippet was a mode I did for someone else. I did not submit that to the codebase because it may not have mass appeal.